src/pages/ProductList/ProductList.js

```diff
@@ -63,30 +84,41 @@ class ProductList extends Component {
                 </div>
                 <div className="styleCategory">
                   <div className="subCategory">
-                    <div data-category="폴로" onClick={this.handleChange}>
+                    <div
+                      className="resetStyleBtn"
+                      data-category=""
+                      onClick={this.styleChange}
+                    >
                       폴로
                     </div>
-                    <div data-category="PARIS" onClick={this.handleChange}>
+                    <div data-category="파리폴로" onClick={this.styleChange}>
                       파리폴로
                     </div>
-                    <div data-category="클래식핏" onClick={this.handleChange}>
+                    <div data-category="클래식핏" onClick={this.styleChange}>
                       클래식핏
                     </div>
                   </div>
                   <div className="subCategory">
-                    <div data-category="레귤러핏" onClick={this.handleChange}>
+                    <div data-category="레귤러핏" onClick={this.styleChange}>
                       레귤러핏
                     </div>
-                    <div data-category="슬림핏" onClick={this.handleChange}>
+                    <div data-category="슬림핏" onClick={this.styleChange}>
                       슬림핏
                     </div>
                   </div>
                 </div>
+
                 <div className="styleFilter">
                   <div className="styleFilterResult">
                     {searchStyle.length}개의 결과
                   </div>
-                  <div className="filterAndAlign">
+                  <div
+                    className="filterAndAlign"
+                    onClick={() => {
+                      this.setState({ filterMoveNum: 0.51 });
+                      this.setState({ grayDisplayNum: 10 });
```

Comment:
```suggestion
                      this.setState({ filterMoveNum: 0.51, grayDisplayNum: 10 });
```
setState는 객체 형태로 한꺼번에 가능합니다

Reply:
🐊 수정 완료하였습니다.